ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -63,7 +63,6 @@
     - [IP DHCP Relay](#ip-dhcp-relay)
     - [IP ICMP Redirect](#ip-icmp-redirect)
     - [LACP](#lacp)
-    - [Link Tracking Groups](#link-tracking-groups)
```

Comment:
already discussed

Reply:
This is the situation because we removed the link for Link Tracking:
![image](https://user-images.githubusercontent.com/88734745/144077316-9aa318a9-b3df-4dcd-9de9-ee1ea98039c5.png)